Therify/directory
# feat: handle Stripe invoice paid webhook

# Description
Adds Webhook listener for Stripe Invoice.paid events
- Feature schema
- Accounts Service methods
- Stripe vendor Entity schema
- Adds placeholder `account/billing` page to launch Stripe customer Portal

# Closes issue(s)
NA
# How to test / repro
Run app and Stripe CLI simultaneously
Stripe CLI: `yarn stripe-cli:listen`
  Copy the returned signing secret to your local env
App:  `yarn dev:doppler`

1. Sign up for account with an email you have access to
2. Pay for plan in onboarding
3. Check DB `plans` table for an`active` plan matching the selection
4. Go to `/providers/account/billing`
5. Launch Stripe Customer portal and login 
6. Change quantity and pay. 
7. Check that the previous `plan`'s status is now `invalidated` and there is a new `active` `plan` record

# Screenshots

# Changes include


# Checklist


# Other comments
